src/Adapter/Presenter/Object/ObjectPresenter.php

```diff
@@ -54,19 +53,8 @@ public function present($object)
             $presentedObject[$fieldName] = $object->{$fieldName};
         }
 
-        if ($object instanceof Product) {
-            $presentedObject['ecotax_tax_inc'] = $object->getEcotax(null, true, true);
-        }
-
```

Comment:
Also doesn't seem to be in scope of PR,does it?

Reply:
Same here - You are right, this is just "saw it, removed it". I can open another PR to clean this up.

Reply:
Moved into separate PR https://github.com/PrestaShop/PrestaShop/pull/29462